bundle-workflow/README.md

```diff
@@ -77,3 +101,22 @@ Signing step (to sign all artifacts):
 ```bash
 ./bundle_workflow/sign.sh artifacts/manifest.yml
 ```
+
+### Plugin Onboarding
+
+1. Update a [manifest](/manifests) for a particular release to include your plugin. We require your plugin name, repository url, and git ref
+that should be used. For unreleased versions this should be a branch in your repository.  Once a release is cut, these refs will be updated
+to build from a tag or specific commit hash.
+
+2. Create a build.sh if you have specific requirements that are not covered by the default script and commit it to your repository. It
+ can live at the root our under scripts/build.sh.
+
+3. Ensure your build.sh reads and passes along both `-Dbuild.snapshot` and `-Dopensearch.version` flags.  Snapshot builds should produce a
```

Comment:
Probably a separate call out to ensure that the format of the output is something like`opensearch-plugin-1.1.0.0.zip` and `opensearch-plugin-1.1.0.0-SNAPSHOT.zip`.